MoviesTVSentiments/app/src/androidTest/java/com/google/moviestvsentiments/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.google.moviestvsentiments;
+
+import android.content.Context;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
```

Comment:
Please do not use wildcard import. https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports